src/lib/libembind.js

```diff
@@ -31,6 +31,10 @@ var LibraryEmbind = {
   $UnboundTypeError: class extends Error {},
   $PureVirtualError: class extends Error {},
   $GenericWireTypeSize: {{{ 2 * POINTER_SIZE }}},
+#if PTHREADS || WASM_WORKERS
+  $postCtors: null, // Unused symbol stub for injecting post ctors call.
+  $postCtors__postset: () => { addAtPostCtor('__embind_post_ctors()'); },
+#endif
```

Comment:
Can't we just make this is `__postset` of some function that a core embind dependnecy?   Adding this new symbol and forcing it to be included seems a more complex than it needs to be.